.github/workflows/release.yaml

```diff
@@ -16,7 +16,9 @@ jobs:
     steps:
       - uses: nearform/optic-release-automation-action@v2
         with:
-          github-token: ${{secrets.github_token}}
+          github-token: ${{ secrets.github_token }}
           npm-token: ${{ secrets[format('NPM_TOKEN_{0}', github.actor)] || secrets.NPM_TOKEN }}
           optic-token: ${{ secrets[format('OPTIC_TOKEN_{0}', github.actor)] || secrets.OPTIC_TOKEN }}
           semver: ${{ github.event.inputs.semver }}
+          build-command: |
+            npm install
```

Comment:
Maybe `npm ci` instead? I never remember all the details, but I recall that there are minor differences regarding interactivity and how the .lock files are treated.

Reply:
Oh, I see! 👍🏻